qiita_pet/templates/private_studies.html

```diff
@@ -134,6 +137,22 @@ <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
             {% end %}
         </tbody>
     </table>
+
+<!--User study Modals-->
+{% for s in user_studies %}
```

Comment:
I thought the abstract was added to the table itself so it could be searched for (which is done above). These divs are representing the modal windows (which are all going to be written into the page).